README.md

```diff
@@ -410,33 +406,8 @@ agenix-rekey = agenix-rekey.configure {
 ## Storage Modes
```

Comment:
I chose to keep this section here in case someone is linking to it, so now it hosts the paragraph that was in the "Installation" section previously.